src/malcolmWidgets/attributeDetails/attributeDetails.component.js

```diff
@@ -74,6 +102,28 @@ const AttributeDetails = props => {
 AttributeDetails.propTypes = {
   attributeName: PropTypes.string.isRequired,
   blockName: PropTypes.string.isRequired,
+  /*
```

Comment:
I think I might have left this as a comment when I was doing performance improvements, could you delete it please